daskernetes/tests/test_objects.py

```diff
@@ -129,3 +129,36 @@ def test_extra_container_config_merge(image_name, loop):
     assert pod.spec.containers[0].args[-1] == "last-item"
 
 
+def test_make_pod_from_dict():
+    d = {
+        "kind": "Pod",
+        "metadata": {
+            "labels": {
+            "app": "dask",
+            "component": "dask-worker"
+            }
+        },
+        "spec": {
+            "containers": [
+            {
+                "args": [
+                    "dask-worker",
+                    "$(DASK_SCHEDULER_ADDRESS)",
+                    "--nthreads",
+                    "1"
+                ],
+                "image": "image-name",
+                "name": "dask-worker",
+                "security_context": {"capabilities": {"add": ["SYS_ADMIN"]},
```

Comment:
This needs to be securityContext, since we're asking for dicts to be in the same format as what Kubernetes expects (those are in camelCase, not _ case).

Reply:
I'm going to spend time fixing and documenting this properly.